src/saml2/client_base.py

```diff
@@ -174,7 +173,7 @@ def __init__(self, config=None, identity_cache=None, state_cache=None,
             "authn_requests_signed": False,
             "want_assertions_signed": False,
             "want_response_signed": True,
```

Comment:
This should change to `False` to allow messages with signed assertions only to be processed. This should also change on the documentation.

```suggestion
            "want_response_signed": False,
```

Reply:
Yes, this is saml2int compliant to change this default. However of course there are some arguments to make that reponse signing is more secure/more defense in depth than assertion signing so there is some case also to make to by default check response signatures and let people change that if they need to.

Reply:
If your IdP signs responses it's probably more secure/defense in depth to not allow this to change. So defaulting to response signing as a requirement might make sense, so only installs that knowingly require only the assertion to be signed will change this?